### ISSUE TITLE
Grouped list accessibility issues.

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1605]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. Can't expand or collapse groups. Right arrow nor left arrow expand or collapse. Space selects as expected.
2. Groups don't announce if they're expanded or collapsed.

**Comment by betrue-final-final** [comment]
In Narrator, it's a little easier to find the issues:

1. Can't expand or collapse groups. Right arrow nor left arrow expand or collapse. Space selects as expected.
2. Groups don't announce if they're expanded or collapsed.

**Comment by betrue-final-final** [comment]
Still an issue.

**Comment by dzearing** [comment]
Kevin can you take a look when possible?

**Comment by KevinTCoughlin** [comment]
> In Narrator, it's a little easier to find the issues:
> 
>     1. Can't expand or collapse groups. Right arrow nor left arrow expand or collapse. Space selects as expected.

Should be tracked via #6318 and requires engineering work. It looks like the spec is:

>  Left arrow: Collapse the group
>  Right arrow: Expand the group
>  Space bar: Select or deselect the group

I will expand on Jason's work and see if a solution is possible.

>     2. Groups don't announce if they're expanded or collapsed.

Should be closed by #6843. Will review and confirm.

Keeping this issue open in the meantime.

**Comment by jdhuntington** [comment]
Closing as dup, given we are tracking the remaining work with #6318
